proto/cosmos/base/grpc/v2/service.proto

```
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package cosmos.base.grpc.v2;
```

buf

Files with package "cosmos.base.grpc.v2" must be within a directory "cosmos/base/grpc/v2" relative to root but were in directory "proto/cosmos/base/grpc/v2".

(PACKAGE_DIRECTORY_MATCH)